### DISCUSSION TITLE
Nominating @brayanhenao for the role of Maintainer

### DISCUSSION THREAD

**Discussion by thitch97** [paketo-buildpacks/feedback#234] [Python Team]
I'd like to nominate @brayanhenao  as a maintainer for the Python subteam. He has a vested interest in the health of the Python buildpacks has made large contributions to the space (e.g. [conda-env-update](https://github.com/paketo-buildpacks/conda-env-update/pull/4)).

👍 / 👎 ?

cc/ @paketo-buildpacks/python-maintainers
@paketo-buildpacks/steering-committee

**Comment by ForestEckhardt** [comment]
Congrats @brayanhenao ! You have been added to the Python Maintainer team.

Please open a PR on the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md) to add @brayanhenao to the list of maintainers.
